# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/tests/crashtests/crashtests.list
# Commit: c7abc15f7444
# Full Hash: c7abc15f7444c9ebc5c6adafc3a1be2d5d1e3090
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2019-09-07 09:40:21
# Description:
#   Bug 1578282 - Use flattened tree parent when getting inserted node container. r=Jamie
#   
#   This assures that a node that has already been inserted in a shadow root
#   subtree does not get erroneously reinserted into the top container.
#   
# ==============================================================================

diff -r 96405b0b1298 -r c7abc15f7444 accessible/tests/crashtests/crashtests.list
--- a/accessible/tests/crashtests/crashtests.list	Fri Sep 06 17:35:48 2019 +0000
+++ b/accessible/tests/crashtests/crashtests.list	Fri Sep 06 00:24:21 2019 +0000
@@ -2,6 +2,7 @@
 load chrome://reftest/content/crashtests/accessible/tests/crashtests/471493.xul
 asserts-if(!browserIsRemote,2) load 884202.html
 load 1572811.html
+load 1578282.html
 load 890760.html
 load 893515.html
 load 1072792.xhtml
